src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+.cards {
+  display: grid;
+  grid-gap: 46px 48px;
+  padding: 50px $space-40;
+}
+
+.card {
```

Comment:
it is better to set padding for entire card rather than add horizontal margins for each element

Reply:
sorry, maybe I didn't understand the comment, this style is used for "cards" element, which is one for the entire card.